liquibase-standard/pom.xml

```diff
@@ -179,6 +179,12 @@
             <artifactId>commons-text</artifactId>
             <version>${commons-text.version}</version>
         </dependency>
+        <dependency>
```

Comment:
@StevenMassaro just to check if any license update is required with adding this artifact?

Reply:
@rberezen the license is a bit different: https://github.com/NanoHttpd/nanohttpd/blob/master/LICENSE.md
But as this is used for testing and we don't ship it then we don't need to update it anywhere.